Mathlib/Mathport/Syntax.lean

```diff
@@ -211,13 +211,8 @@ syntax termList := " [" term,* "]"
 
 /- E -/ syntax (name := applyNormed) "apply_normed " term : tactic
 
-/- B -/ syntax (name := abel) "abel" (ppSpace (&"raw" <|> &"term"))? (ppSpace location)? : tactic
-/- B -/ syntax (name := abel!) "abel!" (ppSpace (&"raw" <|> &"term"))? (ppSpace location)? : tactic
```

Comment:
The new `abel` doesn't have this syntax:
```lean
macro (name := abel) "abel" : tactic => ...
@[inherit_doc abel] macro "abel!" : tactic => ...
```
so this will break mathport. You should move this syntax to `Abel.lean` and make it just ignore or throw when the extra args are used.